docker-compose.yml

```diff
@@ -279,7 +279,10 @@ services:
       - diet-error-handler
     environment:
       << : *govuk-app
+      GOVUK_CONTENT_SCHEMAS_PATH: /govuk-content-schemas
       MONGO_WRITE_CONCERN: 1
+      MONGODB_URI: mongodb://mongo/content-store
+      PORT: 3068
```

Comment:
Sorry the lack of these is past-me's fault 🤦  I have recollections there was method to the madness but it doesn't make sense to today-me.